ui/app/styles/components/tabs.scss

```
@@ -13,20 +13,26 @@
       &:focus {
         box-shadow: none;
       }
-      &.is-active,
-      &.is-active .tab {
+      &.active,
+      .tab {
```

I think we still want to only target active tags here, right? 
```suggestion
      &.active .tab {
```

---

Good catch. I'll amend.